jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -7182,6 +7182,128 @@
 						"description": ""
 					},
 					"response": []
+				}
+			]
+		},
+		{
+			"name": "u) health check application test",
+			"description": "",
+			"item": [
+				{
+					"name": "start jvm with wait true",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "179a918a-9d12-4d3a-9594-17818fefa228",
+								"type": "text/javascript",
+								"exec": [
+									"setTimeout(function(){}, 5000);",
+									"tests[\"Status code is 200\"] = responseCode.code === 200;"
+								]
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"controlOperation\":\"start\"}"
+						},
+						"url": {
+							"raw": "{{url}}/v1.0/jvms/1/commands?wait=TRUE",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"v1.0",
+								"jvms",
+								"1",
+								"commands"
+							],
+							"query": [
+								{
+									"key": "wait",
+									"value": "TRUE",
+									"equals": true
+								}
+							]
+						},
+						"description": ""
+					},
+					"response": []
+				},
+				{
+					"name": "get health check application",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "c2738e9c-ad1d-478c-b7fc-480b5af978a7",
+								"type": "text/javascript",
+								"exec": [
+									"tests[\"Status code is 200\"] = responseCode.code === 200;"
+								]
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {},
+						"url": "https://{{host}}/hct/",
```

Comment:
We shouldn't be expecting the health check application to be deployed. I thought we had a simple application that post man deploys. 

Should these tests be in the health check postman test suite instead?